libs/scripts/release.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# ONLY TO BE RUN ON MASTER
+# master is assumed when using -f and --hard git parameters
+
+set -ex
+
+# Finds the lastest commit that looks like a version commit,
+# and gets the list of commits after that
+function git-changelog () {
+    # Find the latest release commit by using the last time the first `version` key changed
+    version_line_number=$(grep -m 1 -n version package.json | cut -d: -f 1)
+    release_commit=$(git blame -L ${version_line_number},+1 --porcelain -- package.json | awk 'NR==1{ print $1 }')
+
+    # Print the log as "- <commmiter short date> [<commit short hash>] <commit message> <author name>"
+    git log --pretty=format:"- %cd [%h] %s %an" --date=short $release_commit..HEAD
```

Comment:
I changed the format slightly to include date and hash.

The rest is a combination of audius-client, other audius-protocol releases, and Marcus' original script.

Reply:
the changelog doesn't have newlines in it https://github.com/AudiusProject/audius-protocol/commit/6ee58ff21af905ba45f7bfd62be1b91e32a78f10, can we add that for easier reading?

also shameless plug, here's another git log command i wrote (which should have new lines 😛 ) https://github.com/AudiusProject/audius-tooling/blob/c3cc9620a700b97387956f426b3d159a4d91b26e/deploy-scripts/release-version-diff.sh#L45